pkg/cli/alpha/internal/generate.go

```diff
@@ -34,6 +34,7 @@ import (
 	autoupdate "sigs.k8s.io/kubebuilder/v4/pkg/plugins/optional/autoupdate/v1alpha"
 	"sigs.k8s.io/kubebuilder/v4/pkg/plugins/optional/grafana/v1alpha"
 	hemlv1alpha "sigs.k8s.io/kubebuilder/v4/pkg/plugins/optional/helm/v1alpha"
+	hemlv2alpha "sigs.k8s.io/kubebuilder/v4/pkg/plugins/optional/helm/v2alpha"
```

Comment:
```suggestion
	helmv2alpha "sigs.k8s.io/kubebuilder/v4/pkg/plugins/optional/helm/v2alpha"
```

+ the line above

enforcing import-names might help here as well (see https://github.com/kubernetes-sigs/kubebuilder/pull/5058#discussion_r2336743373)

Reply:
Since it is following as it is now, see the v1. We do not need to change it to get this merged; we can work on it in a follow-up if required.